spark-jobs/src/test/scala/za/co/absa/enceladus/standardization/interpreter/StandardizationInterpreter_ArraySuite.scala

```diff
@@ -0,0 +1,218 @@
+/*
+ * Copyright 2018 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.enceladus.standardization.interpreter
+
+import org.apache.spark.sql.types._
+import org.scalatest.FunSuite
+import za.co.absa.enceladus.utils.error.UDFLibrary
+import za.co.absa.enceladus.utils.general.JsonUtils
+import za.co.absa.enceladus.utils.testUtils.{LoggerTestBase, SparkTestBase}
+import za.co.absa.enceladus.utils.implicits.DataFrameImplicits.DataFrameEnhancements
+import za.co.absa.enceladus.utils.schema.MetadataKeys
+import za.co.absa.enceladus.utils.validation.ValidationException
+
+class StandardizationInterpreter_ArraySuite extends FunSuite with SparkTestBase with LoggerTestBase {
+  import spark.implicits._
+
+  private implicit val udfLib: UDFLibrary = new UDFLibrary
+
+  private val fieldName = "arrayField"
+
+  private val errColSchema =  "\n |-- errCol: array (nullable = true)\n"+
```

Comment:
Just a suggestion. Moving test case generators to a separate class (test case factories) can help reuse the test cases in different suites.